lib/ui/widgets/schedule_calendar_tile.dart

```diff
@@ -41,10 +42,12 @@ class _ScheduleCalendarTileState extends State<ScheduleCalendarTile> {
     final courseLocation = courseInfos[1];
     final courseType = courseInfos[2];
     final teacherName = courseInfos[3];
-    final startTime =
-        "${widget.start?.hour ?? '00'}:${widget.start?.minute.toString().padLeft(2, '0') ?? '00'}";
-    final endTime =
-        "${widget.end?.hour ?? '00'}:${widget.end?.add(const Duration(minutes: 1)).minute.toString().padLeft(2, '0') ?? '00'}";
+    final startTime = widget.start == null
+        ? AppIntl.of(widget.buildContext)!.grades_not_available
+        : "${widget.start?.hour}:${widget.start?.minute.toString().padLeft(2, '0')}";
```

Comment:
ça je sais pas exactement comment ça marche en flutter, mais on peut pas forcer qu'on connait que la variable sera pas null dans le ternary?
```suggestion
        : "${widget.start!.hour}:${widget.start!.minute.toString().padLeft(2, '0')}";
```